CHANGELOG.md

```diff
@@ -3,6 +3,50 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 5.0.0-beta.7
+
+_Nov 4, 2021_
+
+- 👁 Move virtualization logic to hook (#3079) @m4theushw
+- 🌍 Add hebrew localization (he-IL) (#3028) @ColdAtNight
+- 📚 Documentation improvements
+- 🐞 Bugfixes
+
+### `@mui/x-data-grid@v5.0.0-beta.7` / `@mui/x-data-grid-pro@v5.0.0-beta.7`
+
+#### Breaking changes
+
+#### Changes
+
+- [DataGrid] Add hebrew localization (he-IL) (#3028) @ColdAtNight
+- [DataGrid] Move virtualization logic to hook (#3079) @m4theushw
+- [DataGrid] Revert year change in the MIT license (#3059) @oliviertassinari
+- [DataGrid] Fix filtering of nullish numeric cells (#3070) @flaviendelangle
+- [DataGrid] Fix selecting 100k rows takes too long (#3077) @m4theushw
+- [DataGrid] Fix `GridEditDateCell` does not handle timezone correctly (#2918) @flaviendelangle
+- [DataGrid] Fix Keyboard navigation broken on page > 0 (#3074) @flaviendelangle
+- [DataGrid] Prevents bubbling in menu header (#3000) @alexfauquette
+- [DataGrid] `setCellMode` is providing the wrong params to `cellModeChange` event (#3025) @flaviendelangle
+
+### Core
+
+- [core] Adapt `useDemoData` for Tree Data (#2978) @flaviendelangle
+- [core] Group update of MUI Core (#3055) @oliviertassinari
+- [core] Ignore `*.tsbuildinfo` files (#3068) @m4theushw
+- [core] Implement tree-based row management (#2996) @flaviendelangle
+- [core] Invert Codesandbox examples on README (#3073) @flaviendelangle
+- [core] Prefix selectors from `useGridContainerProps` with `unsafe` (#3002) @flaviendelangle
+- [core] Reduce `setGridState` and `applyFilters` call when updating `filterModel` (#3023) @flaviendelangle
+- [core] Upgrade monorepo (#3067) @m4theushw
+- [core] Use official MUI repo as monorepo (#3084) @m4theushw
+- [test] Retry each expect until success (#3027) @m4theushw
+- [scripts] Adapt change-log script to github dom modification (#3087) @alexfauquette
```

Comment:
The only tags we use, besides the component name, is "core" or "test" or "docs".

```suggestion
- [core] Adapt changelog script to new GitHub DOM (#3087) @alexfauquette
```